spec/models/lead_provider_spec.rb

```diff
@@ -23,4 +23,12 @@
       end
     end
   end
+
+  describe "normalizing" do
+    subject { FactoryBot.build(:lead_provider, name: " Some lead provider ") }
+
+    it "removes leading and trailing spaces from the name" do
```

Comment:
Do you think it could be worth adding another spec for multiple spaces? 

Something like: 

> it "collapses multiple spaces into one"